### PR TITLE
Add TPS + Remove deprecated methods

### PR DESCRIPTION
**Pull request by Aberdeener**
Title says it all. Tested locally 👍 

Possible todo in the future would be to display 1m, 5m, 15m TPS in an array so we maximize the detail we give the user.

Also, I recommend you take a look at [project lombok](https://projectlombok.org/) as it could help minify this code significantly.

Well done overall!

### CODE REVIEW

**Comment by phybros** [comment]
@Aberdeener if we change the server dep to paper, will it still work on Spigot servers?

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 *.iml
 .idea/
 
+.DS_STORE
```
Comment (phybros) [inline .gitignore]: +1

**Comment by Aberdeener** [comment]
@phybros yep! It'll work still

**Comment by phybros** [comment]
Actually @Aberdeener I think it's got a problem. The `DecimalFormat` part of tps should be taking the `double` from `getTps()[0]` directly instead of it's string value. See here for more deets https://github.com/phybros/servertap/pull/3

**Pull request merged**